apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net.Snippets/GoogleStackdriverAppenderSnippets.cs

```diff
@@ -110,7 +110,86 @@ public void Overview()
                     }
                     Thread.Sleep(1_000);
                 }
-                Assert.False(true, "Log entry failed to appear in StackDriver.");
+                Assert.False(true, "Log entry failed to appear in Google Cloud Logging.");
+            }
+            finally
+            {
+                File.Delete(fileName);
+            }
+        }
+
+        // SampleResource: log4net-custom-labels-template.xml
+
+        [Fact]
+        public void CustomLabels()
+        {
+            DateTime startTime = DateTime.UtcNow;
+            string projectId = _fixture.ProjectId;
+            string logId = _fixture.LogId + $"-{Guid.NewGuid()}";
+            string fileName = "log4net.xml";
+            string resourceName = typeof(GoogleStackdriverAppenderSnippets).Namespace + ".log4net-custom-labels-template.xml";
+
+            string xml;
+            using (var stream = typeof(GoogleStackdriverAppenderSnippets).GetTypeInfo().Assembly
+                .GetManifestResourceStream(resourceName))
+            {
+                using (var reader = new StreamReader(stream))
+                {
+                    xml = reader.ReadToEnd();
+                }
+            }
+            xml = xml.Replace("PROJECT_ID", projectId).Replace("LOG_ID", logId);
+            Assert.False(File.Exists(fileName), "Test would overwrite existing file");
+            File.WriteAllText(fileName, xml);
+
+            try
+            {
+                // Resource: log4net-custom-labels-template.xml log4net_custom_labels_template
+                // Sample: Custom_Labels
+                // Configure log4net to use Google Cloud Logging from the XML configuration file.
+                XmlConfigurator.Configure(LogManager.GetRepository(GetType().GetTypeInfo().Assembly), new FileInfo("log4net.xml"));
```

Comment:
I suspect `GetTypeInfo()` can be removed everywhere now - we only didn't have it in netstandard1.*

Reply:
Ah yes, will remove from all snippets.